readthedocs/proxito/views/serve.py

```diff
@@ -555,4 +556,38 @@ def changefreqs_generator():
 
 
 class ServeSitemapXML(SettingsOverrideObject):
+
     _default_class = ServeSitemapXMLBase
+
+
+class ServeStaticFiles(ServeDocsMixin, View):
+
+    """
+    Serve static files from the same domain the docs are being served.
+
+    AdBlockers and similar don't like calls from external domains,
+    so features like server side search (SSS) don't work, so we serve these
+    files from the same domain instead.
+
+    .. note::
+
+       This can be done at the nginx level too.
+       Serving the static files from django can allow us to disable
+       some features like SSS without users needing to rebuild their docs.
+    """
+
+    def get(self, request, filename):
+        filename = filename.lstrip('/')
+        allowed_files = {
+            'javascript/readthedocs-doc-embed.js',
+            'javascript/readthedocs-analytics.js',
+            'css/readthedocs-doc-embed.css',
+            'css/sphinx_rtd_theme.css',
```

Comment:
Not sure why we serve this one,  but it was in sphinx

https://github.com/readthedocs/readthedocs-sphinx-ext/blob/ea4643d0c204756eefcd32a45b75795686795daf/readthedocs_ext/readthedocs.py#L90